template/runner-config-runners.tpl

```diff
@@ -0,0 +1,63 @@
+[[runners]]
+  name = "${name}"
+  url = "${gitlab_url}"
+  token = "${token}"
+  executor = "${executor}"
+  environment = ${environment_vars}
+  pre_build_script = "${pre_build_script}"
+  post_build_script = "${post_build_script}"
+  pre_clone_script = "${pre_clone_script}"
+  request_concurrency = ${request_concurrency}
+  output_limit = ${output_limit}
+  limit = ${limit}
+  [runners.docker]
+    tls_verify = false
+    image = "${image}"
+    privileged = ${privileged}
+    disable_cache = false
```

Comment:
The auto-scaling docs specifically set `disable_cache = true` so maybe we should too: https://docs.gitlab.com/runner/configuration/runner_autoscale_aws/#the-runnersdocker-section